docs/operators/transform.md

```diff
@@ -0,0 +1,121 @@
+
+Consider the below input schema
+`{FirstName, LastName, Phone, DateOfBirth, Address}`
+
+and output schema
+`{Name, Phone, Age, Address}`
+
+Suppose the requirement as follows:
+    -   "Name" field would be the combination of FirstName and LastName 
+    -   "Age" can be calculated from DateOfBirth.
+            
+Then, below are the expression mappings between output and input fields. 
+
+```
+Name => {$.FirstName}.concat(\" \").concat({$.LastName})
+Age => (new java.util.Date()).getYear() - {$.dateOfBirth}.getYear()
+```
+
+#### Configuration Parameters
+-----------------------------
+
+-   ***expressionMap*** -   Map<String, String>
+    -   Mandatory Parameter
+    -   Specifies the map between the output field and expression using fields of input POJO.
+
+-   ***expressionFunctions*** -   List<String>
+    -   List of imported classes or methods should be made available to expression to use.
+    -   Default Value = {java.lang.Math.*, org.apache.commons.lang3.StringUtils.*, org.apache.commons.lang3.StringEscapeUtils.*, org.apache.commons.lang3.time.DurationFormatUtils.*, org.apache.commons.lang3.time.DateFormatUtils.*}
+
+-   ***copyMatchingFields*** -   boolean
+    -   Specifies whether the matching fields between input and output should be copied as is or not.
+    -   Default Value = true.
+    
+#### Ports
+----------
+
+-   ***input*** -   Port for input tuples.
+-   ***output***    -   Port for transformed output tuples.
+
+#### Attributes
+---------------
+-   ***Input port Attribute - input.TUPLE\_CLASS*** - Fully qualified class name.
+    -   Mandatory attribute
+    -   Type of input tuple.
+
+-   ***Output port Attribute - output.TUPLE\_CLASS*** - Fully qualified class name.
+    -   Mandatory attribute
+    -   Type of output tuple.
+    
+#### Application Example
+------------------------
+
+Please refer [Example] (https://github.com/DataTorrent/examples/tree/master/tutorials/transform) for transform sample application.
+
+### Partitioning of TransformOperator
+Being stateless operator, TransformOperator will ensure built-in partitioners present in Malhar library can be directly by setting few properties as follows:
+
+#### Stateless partitioning of TransformOperator
+Stateless partitioning will ensure that TransformOperator will will be partitioned right at the starting of the application and will remain partitioned throughout the lifetime of the DAG.
+TransformOperator can be stateless partitioned by adding following lines to properties.xml:
+
+```xml
+  <property>
+    <name>dt.operator.{OperatorName}.attr.PARTITIONER</name>
+    <value>com.datatorrent.common.partitioner.StatelessPartitioner:3</value>
+  </property>
+```
+
+where {OperatorName} is the name of the TransformOperator operator.
+Above lines will partition TransformOperator statically 3 times. Above value can be changed accordingly to change the number of static partitions.
```

Comment:
Since you are using a place-holder for the operator name, why not do the same for the partition count as well (e.g. {N}) and change the text accordingly ?